src/freertos_drivers/common/MmapGPIO.hxx

```diff
@@ -0,0 +1,113 @@
+/** \copyright
```

Comment:
I'm not sure that this file belongs here.  Why does not seem specific to FreeRTOS.  Could it not also be used in Linux for example?